xbmc/addons/kodi-addon-dev-kit/include/kodi/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(HEADERS AddonBase.h
+            AudioEngine.h
+            Filesystem.h
+            General.h
+            Network.h
+            kodi_adsp_dll.h
+            kodi_adsp_types.h
+            kodi_audiodec_dll.h
+            kodi_audiodec_types.h
+            kodi_game_dll.h
+            kodi_game_types.h
+            kodi_imagedec_dll.h
+            kodi_imagedec_types.h
+            kodi_inputstream_dll.h
+            kodi_inputstream_types.h
+            kodi_peripheral_dll.h
+            kodi_peripheral_types.h
+            kodi_peripheral_utils.hpp
+            kodi_vfs_dll.h
+            kodi_vfs_types.h
+            kodi_vfs_utils.hpp
+            libKODI_adsp.h
+            libKODI_game.h
+            libKODI_guilib.h
+            libKODI_inputstream.h
+            libKODI_peripheral.h
+            libXBMC_addon.h
+            libXBMC_pvr.h
+            versions.h
+            xbmc_addon_dll.h
+            xbmc_addon_types.h
+            xbmc_audioenc_dll.h
+            xbmc_audioenc_types.h
+            xbmc_epg_types.h
+            xbmc_pvr_dll.h
+            xbmc_pvr_types.h
+            xbmc_vis_dll.h
+            xbmc_vis_types.h)
+
+core_add_library(addons_kodi-addon-dev-kit_include_kodi)
```

Comment:
```
if(NOT ENABLE_STATIC_LIBS)
  core_add_library(addons_kodi-addon-dev-kit_include_kodi)
endif()
```

@ksooo This solves the header only issue. The same needs to done for the other header only libs.